lib/cisco_node_utils/interface.rb

```diff
@@ -806,6 +806,62 @@ def feature_lacp_set(val)
       config_set('interface', 'feature_lacp', state: val ? '' : 'no')
     end
 
+    def load_interval_counter_1_delay
+      return nil if @name[/loop/] || @name[/ethernet.*\S+\.\d+$/]
+      config_get('interface', 'load_interval_counter_1_delay', name: @name)
+    end
+
+    def load_interval_counter_1_delay=(val)
+      fail ArgumentError, 'Interface cannot be sub-intf or loopback' if
+        @name[/loop/] || @name[/ethernet.*\S+\.\d+$/]
+      config_set('interface', 'load_interval_counter_1_delay',
+                 name: @name, delay: val)
+    end
+
+    def default_load_interval_counter_1_delay
+      # for vlan and bdi the default is 60
+      if @name[/(vlan|bdi)/i]
+        config_get_default('interface',
+                           'load_interval_counter_1_delay_vlan_bdi')
+      else
+        config_get_default('interface', 'load_interval_counter_1_delay')
+      end
+    end
+
+    def load_interval_counter_2_delay
+      return nil if @name[/loop/] || @name[/ethernet.*\S+\.\d+$/]
+      config_get('interface', 'load_interval_counter_2_delay', name: @name)
+    end
+
+    def load_interval_counter_2_delay=(val)
+      fail ArgumentError, 'Interface cannot be sub-intf or loopback' if
```

Comment:
I did not add loop for get/set. I think the tests should be clean and readable. If you disagree, I will change it.